auth/server/src/main/java/com/seagates3/parameter/validator/AbstractParameterValidator.java

```diff
@@ -82,6 +82,11 @@ public Boolean isValidListParams(Map<String, String> requestBody) {
                     requestBody.get("Marker"));
         }
 
+        if (requestBody.containsKey("OnlyAttached")) {
```

Comment:
do we have a global static constant file. I can see only attached and other keys are used frequently which can be move there

Reply:
fixed